samples/guestbook/main.go

```diff
@@ -37,6 +37,7 @@ import (
 	"github.com/google/go-cloud/server"
 	"github.com/google/go-cloud/wire"
 	"github.com/gorilla/mux"
+	"go.opencensus.io/exporter/jaeger"
```

Comment:
@ijt just a suggestion since this is a Google Cloud initiated project: if y'all already using Stackdriver, these are the docs for it https://opencensus.io/guides/exporters/supported-exporters/go/stackdriver/#1 and a bunch of other Go exporters as well here https://opencensus.io/guides/exporters/supported-exporters/go/

Reply:
Got it. Based on an offline conversation with @zombiezen I'm going to move this chunk of change to a different issue and pull request, and I'll apply your advice there.